ansible/local_cert_tasks.yaml

```
@@ -9,7 +9,7 @@
         oc get -n {{ namespace }} osnet ctlplane -o json | jq -r .spec.roleReservations.ControlPlane.reservations[0].ip 
       register: get_control_plane_ip_ctlplane
       until: get_control_plane_ip_ctlplane.stdout|trim|ipaddr != False
-      retries: 60
```

This normally only takes a couple of retries. Any idea what is causing such a big delay?

---

when I have seen this the config of the nncp took quiet long and the osnetcfg create took longer then usually.

---

I think I have a better idea. we should probably wait for in the install_networks tasks on  the osnetcfg to have finished before proceed or fail early at that point if there is an issue. that'd make debugging probably easier.